styles/profile_styles.css

```diff
@@ -148,41 +97,104 @@ td:hover span {
     background-color: rgba(0, 0, 255, 0.7);
     border: 2px solid white;
 }
-/* #make_goal {
-    TODO: styles for make goal form.
-    Make anim for appear: page below slide down, new goal form unveiled.   
-}*/
-#challenge_button {
-    /* grid-column: 5 / 8; */
-    /* grid-row: 6 / 7; */
-    margin: 1vw auto;
+
+#make_goal {
+    margin-left: 30%;
+    margin-right: 30%;
+    padding: 5px;
+    border: 2px solid white;
+    background-color: rgba(0, 0, 255, 0.7);
+     overflow: hidden;
 }
-#challenge_button button {
-    background-color: rgba(0, 0, 255, 0.1);
-    border: 1px solid white;
-    color: white;
+#make_goal form {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: repeat(6, 1fr);
+    column-gap: 2%;
+} 
+#make_goal form input,
+#make_goal form textarea {
+    border-radius: 5px;
+}
+#make_goal form #goalDescrip {
+    grid-column: 1 / span 1;
+    grid-row: 1 / span 5;
+}
+#make_goal form #dateStartInput {
+    grid-column: 2 / span 1;
+    grid-row: 1 / span 1;
+    margin-left: 5em;
+}
+#make_goal form #dateStartLabel {
+    grid-column: 2 / span 1;
+    grid-row: 1 / span 1;
+}
+#make_goal form #dateEndInput {
+    grid-column: 2 / span 1;
+    grid-row: 2 / span 1;
+    margin-left: 5em;
+}
+#make_goal form #dateEndLabel {
+    grid-column: 2 / span 1;
+    grid-row: 2 / span 1;
+}
+/* #make_goal fieldset form #amountInput {
+    grid-column: 2 / span 1;
+    grid-row: 4 / span 1;
+    margin-left: 50%;
+}
+#make_goal fieldset form #amountLabel {
+    grid-column: 2 / span 1;
+    grid-row: 4 / span 1;
+} */
```

Comment:
BTW reviews are cool because you can instantly commit (trivial) suggestions:

<img width="460" alt="image" src="https://user-images.githubusercontent.com/13428215/143318347-530a699b-b071-4eb1-8c6d-d909aaa738e3.png">